### ISSUE TITLE
TypeError when checking for time in dimensions

### ISSUE THREAD

**Issue by Manangka** [Deltares/imod-python#37]
In GitLab by @jdelsman on Feb 28, 2019, 13:51

_has_dim throws an error:
TypeError: cannot compare cftime._cftime.DatetimeProlepticGregorian(1911, 4, 10, 0, 0, 0, 0, -1, 1) and None

imod 0.5.0
cftype '1.0.0b1'

**Comment by Manangka** [comment]
In GitLab by @jdelsman on Feb 28, 2019, 13:53

it is caused by the 'if None in seq', cftime == None is not allowed.

The following works:  (replaced by a cftime is None comparison)

```
def _has_dim(seq):
    """Check if a either 0 or all None are allowed

    Returns
    -------
    True if no None in seq, False if all None, error otherwise
    """
    comp = [x is None for x in seq]
    if any(comp):
        if all(comp):
            return False
        else:
            raise ValueError("Either 0 or all None allowed")
    return True
```

**Comment by Manangka** [comment]
In GitLab by @visr on Feb 28, 2019, 17:26

mentioned in commit 86e472b88e36d2c714d98d1b0534324e3f004434

**Comment by Manangka** [comment]
In GitLab by @visr on Feb 28, 2019, 17:29

I cannot reproduce, likely because I am using a newer cftime? Would be good to at least upgrade to the 1.0 release, not a beta version.
```
python 3.7
imod 0.5.0
cftime 1.0.3.4
```

Still fine to work around this, see !17

**Comment by Manangka** [comment]
In GitLab by @jdelsman on Mar 1, 2019, 11:28

closed via merge request !17

**Comment by Manangka** [comment]
In GitLab by @jdelsman on Mar 1, 2019, 11:28

mentioned in commit 57e4825631db0da6eaeffc76eec7db9e3615e5c6

**Comment by Manangka** [comment]
In GitLab by @visr on Mar 1, 2019, 11:28

closed via commit 86e472b88e36d2c714d98d1b0534324e3f004434
